diff --git a/build/lib/electron.ts b/build/lib/electron.ts
index 016c25f7..597dc71d 100644
--- a/build/lib/electron.ts
+++ b/build/lib/electron.ts
@@ -107,15 +107,15 @@ export const config = {
 	version: electronVersion,
 	tag: product.electronRepository ? `v${electronVersion}-${msBuildId}` : undefined,
 	productAppName: product.nameLong,
-	companyName: 'Microsoft Corporation',
-	copyright: 'Copyright (C) 2026 Microsoft. All rights reserved',
+	companyName: 'DBCode Wrapper',
+	copyright: 'Copyright (C) 2026 DBCode Wrapper contributors',
 	darwinExecutable: product.nameShort,
 	darwinIcon: 'resources/darwin/code.icns',
 	darwinBundleIdentifier: product.darwinBundleIdentifier,
 	darwinApplicationCategoryType: 'public.app-category.developer-tools',
-	darwinHelpBookFolder: 'VS Code HelpBook',
-	darwinHelpBookName: 'VS Code HelpBook',
-	darwinBundleDocumentTypes: [
+	darwinHelpBookFolder: 'DBCode Wrapper HelpBook',
+	darwinHelpBookName: 'DBCode Wrapper HelpBook',
+	darwinBundleDocumentTypes: product.darwinBundleDocumentTypes ?? [
 		...darwinBundleDocumentTypes({ 'C header file': 'h', 'C source code': 'c' }, 'c'),
 		...darwinBundleDocumentTypes({ 'Git configuration file': ['gitattributes', 'gitconfig', 'gitignore'] }, 'config'),
 		...darwinBundleDocumentTypes({ 'HTML template document': ['asp', 'aspx', 'cshtml', 'jshtm', 'jsp', 'phtml', 'shtml'] }, 'html'),
